src/components/cart/OrderSummary/order-summary.scss

```diff
@@ -0,0 +1,24 @@
+@import "../../../styles/scaffold";
+
+.order-summary[data-order-summary] {
+  padding: 1rem;
```

Comment:
We are using our css tokens variables to style. You can find it on `styles/globals.scss`